src/render/ScreenCreditController.js

```diff
@@ -18,74 +18,58 @@
  */
```

Comment:
@pdavidc Should we leave ScreenCreditController in the render folder given that it is now a layer? Given that it is the first of WWW's internal layers, perhaps putting it in the layer folder sends the wrong message.

Reply:
I agree with your thinking @markpet49. At the moment, the layer folder is intended for application focused layers.